src/worker/ghidra/sleighMishegos.cc

```diff
@@ -77,6 +74,11 @@ void SleighMishegos::initialize(DocumentStorage &store)
     restoreXml(el);
   } else
     reregisterContext();
+
+  pos = new ParserContext(cache);
+
+  // Values taken from DisassemblyCache::initialize
+  pos->initialize(75, 20, getConstantSpace());
```

Comment:
Initialization of the `ParserContext` here fixes the heap buffer overflow 👍

Reply:
Comment says that the values were taken from another function, but is there an explanation for them? Fine if not, but if so we should copy that explanation here.

Reply:
No reason that I can see. I made a comment to clarify that these values were copied from upstream 👍